.react.eslintrc.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
This file was deleted and do not need anymore

Reply:
opened another PR to delete this file: https://github.com/DevExpress/devextreme-demos/pull/1282